features/citizens/citizens.feature

```diff
@@ -278,27 +278,21 @@ Feature: Citizen journey
       Then I should be on a page showing 'Check your answers'
 
   @javascript
-  Scenario: I have one young dependant
+  Scenario: I have one baby dependant and one 16 years old dependant
     Given An application has been created
     Then I visit the first question about dependants
     Then I should be on a page showing "Do you have any dependants?"
     Then I choose "Yes"
     Then I click 'Save and continue'
     Then I should be on a page showing "Enter your first dependant's details"
-    Then I fill "Name" with "John Doe"
+    Then I fill "Name" with "baby Doe"
```

Comment:
Surely "John Doe Jr." :smile:

Reply:
haha.
But there is also `teenager Doe`.
I chose to name them like this to emphasise that one is less than 15 years old and the other one is between 16 and 18.

Do you know of any famous families we could use?
I wish the Simpsons had one teenager...

Reply:
The only ones I can come up with are:

- Pugsley Addams (pre-teen)
- Wednesday Addams (18 in the Musical)